HW2.R

```diff
@@ -0,0 +1,76 @@
+#########
+###HW2###
+#########
+
+read.csv("iris.csv") #read in data set
```

Comment:
You have to save the object returned by read.csv() by giving it a name, e.g:
iris <- read.csv("Iris.csv") 
It works for this because iris is a built in dataset, but normally it wouldnʻt be available in your workspace.